src/main/java/core/basesyntax/WorkWithFile.java

```diff
@@ -1,7 +1,67 @@
 package core.basesyntax;
 
+import java.io.BufferedReader;
+import java.io.BufferedWriter;
+import java.io.FileReader;
+import java.io.FileWriter;
+import java.io.IOException;
+
 public class WorkWithFile {
+    private static final String SPLIT_SYMBOL = " ";
+    private static final String SPLIT_COMMA = ",";
+    private static final String OPERATION_TXT_0 = "supply";
+    private static final String OPERATION_TXT_1 = "buy";
+    private static final String OPERATION_RESULT = "result";
+    private static final int OPERATION = 0;
+    private static final int AMOUNT = 1;
```

Comment:
I would recommend to change name of these variables to more informative
(ex `OPERATION_SUPPLY`, `OPERATION_BUY`, `OPERATION_INDEX`, `AMOUNT_INDEX`)

Reply:
thank you a lot